qutip/core/cy/qobjevo.pxd

```diff
@@ -0,0 +1,23 @@
+#cython: language_level=3
+
+from qutip.core.data cimport CSR, Dense, Data
+from qutip.core.data.base cimport idxint
+
+cdef class QobjEvo:
+    cdef:
+        list elements
+        set funcQEvo
```

Comment:
Unrelated to this PR, but why is `funcQEvo` a set?

Reply:
I think it actually just needs removing from the definition file - it already was removed from the implementation.